src/components/AppLayout/index.tsx

```diff
@@ -92,6 +93,7 @@ const Layout: React.FC<Props> = ({
 }): React.ReactElement => {
   const [mobileNotSupportedClosed, setMobileNotSupportedClosed] = useState(false)
   const { pathname } = useLocation()
+  const { onClickInstall } = useMobileInstallHandler()
```

Comment:
Why is this needed?

Reply:
To separate getting the install prompt from the component logic.